src/setup-k8s/templates/fabric-k8s/scripts/util.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+
+retry()
+{
+        local n=0
```

Comment:
please pay attention to our standard formatting of .sh files. You can reformat them with `shfmt -i=2 -l -w directory-name`